scripts/apine.py

```diff
@@ -0,0 +1,51 @@
+#!/usr/bin/env python
+#
+# MIT License
+# Copyright (c) INCF
+#
+# apine.py
+# Utility for interacting with Apine objects
+# Created 10/12/2017 initially by G. Kiar
+#
+# To install dependencies, in a fresh virtual environment run:
```

Comment:
- for people not familiar with virtualenvs that's a rather short description :)
- may be worth adding a requirements.txt later for this

Reply:
Agreed - when we package it, we should take care of all of this; current iteration was more meant to be useful for us :)